src/pages/community/patterns/generate-an-api-key/index.mdx

```diff
@@ -15,6 +15,8 @@ When designing for the generation of API keys, ensure users know what the key is
 
 ![Example of an API key being generated](images/1.png)
```

Comment:
@chloepoulterdesign, it would be good to add a caption here. Something like "Example of... " That way the sentence below referencing the storybook will be appropriate spaced and not mistaken for the caption. Thank you!